src/app/ApplicationTemplate.Mobile/iOS/Info.plist

```diff
@@ -0,0 +1,62 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+	<dict>
+		<key>CFBundleDisplayName</key>
+		<string>ApplicationTemplate</string>
+		<key>CFBundleIdentifier</key>
+		<string>com.nventive.internal.applicationtemplate</string>
+		<key>CFBundleShortVersionString</key>
+		<string>1.0</string>
+		<key>CFBundleVersion</key>
+		<string>1.0</string>
+		<key>LSRequiresIPhoneOS</key>
+		<true/>
+		<key>UIDeviceFamily</key>
+		<array>
+			<integer>1</integer>
+			<integer>2</integer>
+		</array>
+		<key>UILaunchStoryboardName</key>
+		<string>LaunchScreen</string>
+		<key>UIRequiredDeviceCapabilities</key>
+		<array>
+			<string>armv7</string>
```

Comment:
We shouldn't need to support armv7 anymore. Our iOS min target version is anyway to high for armv7 devices. See [here ](https://www.innerfence.com/howto/apple-ios-devices-dates-versions-instruction-sets)for more info.